exdb/templates/exdb/experience_approval.html

```diff
@@ -82,8 +33,8 @@
             {% endfor %}
         </table>
         <div>{% trans "Add Comment" %}:</div>
-        <span class="form-errors">{{ form.message.errors }}</span>
-        <div>{{ form.message }}</div>
+        <span class="form-errors">{{ comment_form.message.errors }}</span>
```

Comment:
Since `message.errors` outputs a `<ul>` tag, this `<span>` should be changed to a `<div>`.